src/cluster.h

```diff
@@ -40,10 +40,17 @@ namespace zim
 
       const CompressionType compression;
       const bool isExtended;
-      Offsets offsets;
       std::shared_ptr<const Reader> reader;
+
+      // offset (relative to the ZIM file) of the first blob of this cluster
```

Comment:
No, this is the offset relative to the reader of the first blob of the cluster.
Or, seen otherwise, relative to the start of the whole cluster data (if the cluster is compressed, this is the offset relative from the start of the uncompressed data)

Reply:
Indeed. Good that you caught that mistake now. Otherwise the documentation would serve the opposite of the intended purpose.